src/models/movies-model.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+const db = require('../db');
+
+class MoviesModel {
+
+  async getList() {
+      const query = {
+        text: 'SELECT * FROM movie ORDER BY year ASC;',
+        values: [],
+      }
+      const res = await db.query(query);
+
+      return res.rows;
+  }
+
+  async getMovieById(id) {
+    const query = {
+      text: 'SELECT * FROM movie WHERE id = $1;',
+      values: [id],
+    };
+    const res = await db.query(query);
+
+    return res.rows[0];
+  }
+
+  async getMoviesTitles(year) {
```

Comment:
For my mind will be better to use `getList` here(and add year filter to the `getList`)

Reply:
> For my mind will be better to use `getList` here(and add year filter to the `getList`)

In this case, I must sort titles in Express. I'm not sure but sorting in Postgresql should be faster and more productive than in Express. Isn't it?

Reply:
agree, this is the reason why I say
> and add year filter to the `getList`

Means, you should filter by DB

Reply:
> agree, this is the reason why I say
> 
> > and add year filter to the `getList`
> 
> Means, you should filter by DB

ok, refactored